examples/gitlab-centos7-atomicapp/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM projectatomic/atomicapp:0.1.3
+
+MAINTAINER Navid Shaikh <nshaikh@redhat.com>
+
+LABEL io.projectatomic.nulecule.specversion="0.0.2" \
+      io.projectatomic.nulecule.providers="kubernetes"\
+      Build="docker build --rm --tag swordphilic/gitlab-atomicapp ."
```

Comment:
We don't have any "official" usage for it, but it's probably nice to have if you have some fancy bash aliases set up (https://github.com/vpavlin/docker-tools/blob/master/bash_aliases_docker#L65)